Tasks/AzureFunctionOnKubernetesV0/package-lock.json

```diff
@@ -74,33 +64,17 @@
       }
     },
     "azure-pipelines-tool-lib": {
-      "version": "0.12.0",
-      "resolved": "https://registry.npmjs.org/azure-pipelines-tool-lib/-/azure-pipelines-tool-lib-0.12.0.tgz",
-      "integrity": "sha512-JAlFvMTtEXISrnJY/kgq0LecLi089RqXRf/gMsXYbflmzszklkc+LUJpR0A7NDmJ+9/MWpKY/ZX+Q/zirYa7gw==",
+      "version": "0.11.0",
```

Comment:
Why's the version downgraded here? 
Undo these changes

Reply:
Done